templates/repo/search.tmpl

```diff
@@ -20,12 +20,9 @@
 		</div>
 		{{if .CodeIndexerUnavailable}}
 			<div class="ui error message">
-				<p>{{ctx.Locale.Tr "repo.search.code_search_unavailable"}}</p>
+				<p>{{ctx.Locale.Tr "search.unavailable" (ctx.Locale.Tr "search.kind.code")}}</p>
 			</div>
 		{{else if .Keyword}}
-			<h3>
-				{{ctx.Locale.Tr "repo.search.results" .Keyword .RepoLink .RepoName}}
-			</h3>
```

Comment:
?

Reply:
I don't see a reason for this to be there. In every other case we simply output the search results, but with code search we first print this header containing the search term (already visible in the search field above) and the repo (should be obvious)